examples/apps/simple_imaging_app/gaussian_operator.py

```diff
@@ -25,12 +25,20 @@ class GaussianOperator(Operator):
     """
 
     def compute(self, op_input: InputContext, op_output: OutputContext, context: ExecutionContext):
+        import numpy as np
         from skimage.filters import gaussian
         from skimage.io import imsave
 
         data_in = op_input.get().asnumpy()
         data_out = gaussian(data_in, sigma=0.2, channel_axis=2)  # Add the param introduced in 0.19.
 
+        # Make sure the data type is what PIL Image can support, as the imsave function calls PIL Image fromarray()
+        # Some details can be found at https://stackoverflow.com/questions/55319949/pil-typeerror-cannot-handle-this-data-type
+        print(f"Data type of output: {type(data_out)!r}, max = {np.max(data_out)!r}")
```

Comment:
Any reason not to use logging fx?

Reply:
Good point, though this had been created as a simple example for illustration purpose only, not part of the SDK at all.